src/SFA.DAS.EmployerIncentives.Functions.LegalEntities/Services/LegalEntities/IVendorRegistrationFormService.cs

```diff
@@ -1,4 +1,6 @@
-﻿using System;
+﻿using SFA.DAS.EmployerIncentives.Functions.LegalEntities.Services.LegalEntities.Types;
+using System;
+using System.Collections.Generic;
```

Comment:
is this needed?

Reply:
Hangover from when I added a new service method, which was then moved into the inner API domain handlers instead. Removed.